.github/workflows/ci.yml

```diff
@@ -111,15 +111,15 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-20.04
+          - ubuntu-22.04
         features: ["", "--features pro", "--all-features"]
         build: ["", "--release"]
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-      - name: Add UbuntuGIS ppa
-        run: sudo add-apt-repository ppa:ubuntugis/ppa
+      # - name: Add UbuntuGIS ppa
+      #  run: sudo add-apt-repository ppa:ubuntugis/ppa
```

Comment:
Which GDAL version is used here?

I guess you need to remove it if this is no necessary anymore.
Moreover, you have to adapt the readme.

Reply:
Ubuntu 22.04 ships 3.4.1 as you can see here: https://packages.ubuntu.com/jammy/gdal-bin
Maybe we should gp kach to ubuntu gis at a later point. Ubuntu-gis unstable ships 3.4.3 as you can see here: https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable?field.series_filter=jammy
There is no ubuntu gis stable for 22.04 (yet).

Reply:
[7674a71](https://github.com/geo-engine/geoengine/pull/648/commits/7674a71878fa2c7964cd2e3e8d13ef6ba12bdb7b)